docs/getting_started/slambox_setting.md

```diff
@@ -134,7 +133,48 @@ $ETH_PORT 25180#
 $ETH_IP 192.168.1.50#
 ```
 
-### SLAMBOX Mavlink Configuration {#SLAMBOXSettingsWithSerial-ConfigureSLAMBOX-SLAMBOXMavlinkConfiguration}
+### Set Static IP for Ethernet Communication with SLAMBOX {#SLAMBOXSetting-ConfigureSLAMBOX-SetStaticIPforEthernet}
+
+- Using OTG port of SLAMBOX, then make your device's (pc or others) ip to static.
+
+- Type the commands below in your terminal
+
+```bash
+# Check the SLAMBOX device with ifconfig
+ifconfig
+
+enx1632cf7fcfc0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
```

Comment:
`enx1632cf7fcfc0` should be `eth0` and let the reader know that they need to change `eth0` by their device name.

Another sugesstion
  - Show them GUI screenshots on how to change IP address.
  - People prefers to do in CLI probably already know what to do.